### PR TITLE
Add michaels.com

### PR DESCRIPTION
**Pull request by Rabadash8820**
Add the arts & crafts store, Michaels.

Here is the account settings page for updating 2FA:

image

### CODE REVIEW

File: entries/m/michaels.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Michaels": {
+    "domain": "michaels.com",
+    "img": "michaels.com.png",
```
Comment (hkamran80) [inline entries/m/michaels.com.json]:
```suggestion
```

Here's an SVG you can use:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 160 40"><path fill="#CF1F2E" fill-rule="evenodd" d="M106 22.2c-.4 1.2-1.5 1-1.7.5l-.4-.5-.4-.3c-2.6-2.7-5.2-.4-7 1.6-2.8 3.2-6.2 14-2.5 12.1 3-1.4 5.7-5.8 7.5-8.7l1.2-2c1.6-2.2 4.3-1.2 3.9 1.5l-.1.4c-.3 1.5-.8 4.5-.4 6.6.4 2.2 2.7-.5 3.7-1.8l.3-.3c-.5-3.4 0-6.9 1.7-9.9 4-7.3 12-5 10 1.4-.9 3-3.2 5.5-6.3 6.6-1.2.4-2-.8-.7-1.8 1.6-1 2.8-2.3 3.6-4 1.1-2-.2-3.8-2.3-2.3-3 2.2-4.5 12 .1 14.8 4.5 2.3 8-2.3 11.1-7.9-1.6-6.4-1.3-17.8 3.3-23.6 4.8-6.1 11.6-3.7 11.7.8.2 6.1-4.8 14.7-7 18.6-.6 1-1 1.6-1.1 2-.5 1.1-2.3.6-1.5-1.2a37.5 37.5 0 0 0 6-19.6c-.5-3.1-3-2.4-5.6.3-5.9 8-3.2 20-1.4 27l.9 2.7c.3 1 .6 1.8.6 2.2-.2 2.3-2.2 2-3 .2a25 25 0 0 1-2-5.4c-4.6 7.3-10.9 7.8-14.4 5.4a10 10 0 0 1-3-3.4c-4.9 4.5-9 1.4-9-3.3l-.3.3c-.8 1.1-3.4 4.6-6.7 6.5-3.1 1.8-5.8.7-6-2.1-.4-5.4 5-12.8 6.9-14.8 2-2.2 4-3.6 6.7-3.5 2.7 0 4 3.5 3.5 4.9ZM46 33.9a19.5 19.5 0 0 1 2.8-14.4c.8-.7 1.9-.8 2.7-.3a2 2 0 0 1 1 2.4c-.2 1-.6 2-1 2.9-1.1 3-1.7 6.2-1.7 9.4.4 4.8 2.3 3.3 2.9 2.8.4-.3 2-1.9 2.5-5.6.5-2.8 2.2-9.4 8-12.2 4-2 6.9.2 7 2.7 0 2.4-2.8 2-3.5 1.1v-.2c-.5-.5-1.1-1.4-2.2-1-2.2 1-5.2 5.6-5.1 10.8 0 5.2 5 4.2 7.8 2 3-2.1 4.8-7 5-8.2.4-6.9 5-21.6 7.3-24 2.5-2.5 4.7 0 4.2 1l-2.1 4.8a59.8 59.8 0 0 0-6.8 23.3C76.9 25 81 20.1 85.5 19.5c4-.6 4.7 2.2 3.7 4.5l-.5 1.2A17.8 17.8 0 0 0 87 35.4c.1 1.3.3 3-.4 3.4-1.9 1.1-2.7-1.2-2.8-3.8 0-2.2 1.2-6.8 1.8-8.6l.1-.8c1-3.9-2.7-2.4-3.7-1.3-3.7 3.7-5.5 9.7-6.2 12l-.2.8c-.5 1.7-2.7.9-3.1-1a18 18 0 0 1-.6-4c-4.7 6.6-13.5 9-16 3.7-4.5 5.3-8.6 3.8-10-1.9Zm-3.2.6c-.8-3-.3-9.4.2-13 .4-3.2 1-5.6 1.7-7.8.6-2.4 1.1-4.5 1.1-6.6 0-1.9-.6-5.8-4.7-6C30.7.9 26.9 25.6 26.4 29l-.2-4c0-3.7-.2-9-.7-13.4-1-9.8-3.2-10.3-5.4-10.5-7-.5-8.1 11.7-8.2 24.4 0-2.8-.5-6-2.3-8-1.4-1.5-3.5-1.3-2 .6 0 .1 0 .3.2.5l.3.5a33 33 0 0 1 2.3 10.6c.5 4.8 1 9.1 2.5 9 .9 0 2.2-.3 2-3.9-.3-10.7.5-29.9 4.3-30 2.7 0 3.1 9.2 3.5 17.2.2 3.7.3 7 .7 9.4.6 3.8 4.6 4 5.9-1.3.4-1.9.8-3.6 1-5.2 1-4.6 1.7-8.5 4.3-14 4.8-10 8.8-6.6 7 .3l-.3 1.3c-1.1 4-3.3 11.8-2 20.5.5 2.9 2.3 6.5 3.7 5.7.8-.4.5-1.7-.1-3.8l-.1-.4ZM148.1 20c-.7-2.4-6.3-1.3-5 1.7.4 1 1.8 1.9 3.4 2.9 2 1 4 2.4 5.4 4.3 4 5.8-4.8 12.3-11.5 9.2-4.7-2.2-4.8-5.6-4.4-7 .4-2 2.7-2 3-1 .3 1.3 0 1.7-.2 2l-.2.7c.4 3.3 5.8 4.7 8.7 1.8 4-3.9.5-6.4-.2-7l-2.2-1.1a9 9 0 0 1-3.8-2.6c-3.5-4.7 1.9-8.8 5.9-7.6 1.8.5 3.3 1.8 3 3.5-.4 1.4-1.5 1.3-1.9.2ZM53 12a3 3 0 0 0-2.2 1.6 3 3 0 0 0 0 2.7 2 2 0 0 0 2.5.8 4.3 4.3 0 0 0 2.3-3.1c0-1.3-.8-2.2-2.6-2Zm98.5 4.1h.1c.2 0 .3 0 .3.2l-.3.1h-.1v-.3Zm.1.5v.4h-.3v-1h.8v.2c0 .2 0 .3-.2.3.2 0 .2.1.2.3l.1.2h-.2l-.1-.2c0-.2-.1-.2-.3-.2Zm.1-1c.4 0 .7.2.9.5.2.4 0 .8-.2 1a1 1 0 0 1-1 .2.9.9 0 0 1-.6-.8c0-.3.1-.5.3-.6l.6-.3Zm-1 .3a1.1 1.1 0 0 1 2 0c.2.4.2.8 0 1.1-.2.3-.6.6-1 .6s-.8-.3-1-.6a1 1 0 0 1 0-1Z"/></svg>
```

File: entries/m/michaels.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "categories": [
+      "retail"
+    ]
```
Comment (hkamran80) [inline entries/m/michaels.com.json:14]:
```suggestion
    ],
    "regions": [
      "ca",
      "us"
    ]
```

Their [corporate about page](https://www.michaelspressroom.com/about) says they operate stores in the U.S. and Canada.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/m/michaels.com.json (hunk 2 above)
Comment (Rabadash8820) [reply entries/m/michaels.com.json:14]: Good catch!

File: entries/m/michaels.com.json (hunk 1 above)
Comment (Rabadash8820) [reply entries/m/michaels.com.json]: Nice, updated in f11c40575cb759bc5903a4cb4c494d56aefd1c7e. Out of curiosity, did you find that SVG or generate it somehow?

Comment (hkamran80) [reply entries/m/michaels.com.json]: I found it on their website in the Organization JSON-LD.

**Review by hkamran80 — APPROVED** [review]

File: entries/m/michaels.com.json (hunk 1 above)
Comment (Rabadash8820) [reply entries/m/michaels.com.json]: Oh wow, I've never heard of JSON-LD; this is an awesome source of info! FYI, this image is a wider image with the whole "Michaels" name; the PNG that I originally included is just their square "M" logo (used for the site's favicon). Not sure if you wanted all site images to be square.

**Pull request merged**